### DISCUSSION TITLE
Custom error messages for Superstructure validation schemas?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [react-hook-form/react-hook-form#8725] [Q&A]
I want to use a Superstructure schema to validate my form with the @hookform/resolvers package, but I'm not sure how to hook up custom error messages to it. How do I do this?

**Comment by Moshyfawn** [comment]
It's more of a superstruct question than RHF. Take a look at ianstormtaylor/superstruct#568, more specifically, at [what some of their users have come up with](https://github.com/ianstormtaylor/superstruct/issues/568#issuecomment-808872095) to show the user-facing validation errors.
